### PR TITLE
Added documentation for Custom Connector

### PR DESCRIPTION
**Pull request by negyxo**
Here is a draft version for Custom Connector. Please review it and change it accordingly.

### CODE REVIEW

File: src/guides/v7/sites_frameworks/getting_started/custom_connector.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+Select `Web app hosting` and Custom Connector will be installed automatically and after that you will be prompted to enter Custom Connector configuration options:
+
+```bash
+✔ What is the build directory of your app? … dist
+✔ What is the entry file of your app … index.js
+✔ What is the envirovment variable name for port (if it is available)? … PORT
+✔ Please specify build command if exists … custom_fw build
+✔ Please specify dev command if exists … custom_fw dev
+✔ Please specify dev ready message or timeout value in seconds to wait › 5
+```
```
Comment (negyxo) [inline src/guides/v7/sites_frameworks/getting_started/custom_connector.md:48]: I am not sure how to display CLI output, bash seems to color some words (it recognise them as commands probably)

**Comment by RaeesBhatti** [comment]
@tristanlee85 Please take this over, make any changes if neccessary and publish when you think it's ready.

**Comment by RaeesBhatti** [comment]
@tristanlee85 @negyxo Please update this to "Node.js Connector" and `edgio/nodejs-connector` names.

**Comment by tristanlee85** [comment]
> @tristanlee85 @negyxo Please update this to "Node.js Connector" and `edgio/nodejs-connector` names.

I've updated the guide to anticipate the name change of the package.

**Comment by tristanlee85** [comment]
Actually I need to revert this rename change and deploy these docs since I see the core package was already released under "custom connector" and no documentation to point to.

**Pull request merged**